### ISSUE TITLE
Use deprecation package

### ISSUE THREAD

**Issue by mdbenito** [appliedAI-Initiative/pymetrius#9]
Make it a dependency, and check in the release script that no deprecated methods are left behind in a release. [deprecation](https://deprecation.readthedocs.io/en/latest/) allows marking the moment something is tagged for deprecation and the release at which it will be removed, e.g.
```python
@deprecated(
    deprecated_in="0.6.0",
    removed_in="0.7.0",
    details="This class has been renamed. Use StandardErrorRatio instead",
)
class StandardError(StandardErrorRatio):
    pass
```

**Comment by MischaPanch** [comment]
I don't think this needs to be part of a template. Users can add it themselves if they want to, right?

**Comment by mdbenito** [comment]
Maybe you are right. My reasoning was that if pymetrius is for libraries, then every instantiation of pymetrius will need this package because careful deprecation is a must for every library. It took me a while to realize that the other only alternative, pydeprecate, was not good enough (for reasons I have already forgotten), so I thought it might save others some time to know about it.
